addon/data/panel.css

```diff
@@ -79,6 +79,12 @@ input[type=text]:focus {
   overflow: hidden;
 }
 
+.alpha-badge {
+  position: absolute;
+  top: 0px;
+  left: 0px;
+}
```

Comment:
I think this should have `pointer-events: none` too, so it can't overlap any functional element.  Or it could be a background image.